collector/src/execute.rs

```diff
@@ -187,6 +188,8 @@ impl Profiler {
                 | Profiler::Callgrind
                 | Profiler::Dhat
                 | Profiler::DhatCopy
+                | Profiler::Bytehound
+                | Profiler::Massif
```

Comment:
Best to be consistent with the ordering, here and in other `match` statements. Putting Bytehound always after Massif seems reasonable.